pom.xml

```diff
@@ -132,6 +134,67 @@
+                    <configuration>
+                        <jreleaser>
+                            <project>
+                                <copyright>2020-Present Dice Parser</copyright>
+                            </project>
+                            <release>
+                                <github>
+                                    <owner>${github.org}</owner>
+                                    <name>${github.project}</name>
+                                    <tagName>dice-parser-{{projectVersion}}</tagName>
+                                    <uploadAssets>NEVER</uploadAssets>
+                                    <changelog>
+                                        <formatted>ALWAYS</formatted>
+                                        <labelers>
+                                            <labeler>
+                                                <!-- mark all dependency updates -->
+                                                <label>deps</label>
+                                                <title>regex:Bump </title>
+                                            </labeler>
+                                        </labelers>
+                                        <categories>
+                                            <category>
+                                                <!-- Change name and order of uncategorized items -->
+                                                <title>General Updates</title>
+                                                <key>&lt;&lt;UNCATEGORIZED&gt;&gt;</key>
+                                                <order>1</order>
+                                                <labels>
+                                                    <label>uncategorized</label>
+                                                </labels>
+                                            </category>
+                                            <category>
+                                                <!-- Move dependency updates to bottom of change log -->
+                                                <title>Dependency Updates</title>
+                                                <key>deps</key>
+                                                <order>999</order>
+                                                <!-- bot commits, don't include author -->
+                                                <format>- {{commitShortHash}} {{commitTitle}}</format>
+                                                <labels>
+                                                    <label>deps</label>
+                                                </labels>
+                                            </category>
+                                        </categories>
+                                        <hide>
+                                            <!-- uncategorized changes have been "moved" to General Updates -->
+                                            <uncategorized>true</uncategorized>
```

Comment:
@aalmiray If you get a sec, can you take a quick look at this?
This seems to work fine locally, but I also feel like I'm hacking around the intent here.

Basically, I'm moving anything that looks like a dependency update to the bottom, and re-labeling anything uncategorized as "General Updates".

Example output from running `mvn jreleaser:changelog`

```md
## Changelog

## General Updates
- 810429b Add jreleaser changelog config (Brian Demers)
- 8625a50 Update version to next snapshot release (Brian Demers)
- 382e052 Make build reproducible (#48) (Brian Demers)
- 3c7c8a9 Update kotlin to 1.8.10 (#47) (Brian Demers)

## Dependency Updates
- f299be6 Bump actions/checkout from 3.1.0 to 3.3.0 (#45)
- 675bdbb Bump maven-compiler-plugin from 3.10.1 to 3.11.0 (#46)


## Contributors
We'd like to thank the following people for their contributions:
Brian Demers
```

Also note,  `JRELEASER_GITHUB_TOKEN` is a required property by this goal, but that value is not used, should I open an issue for that?